src/api/vectordb/upsert.rs

```diff
@@ -40,7 +40,11 @@ pub(crate) async fn upsert(
         let _ = run_upload(ctx.into_inner(), vec_store, convert_vectors(body.vectors));
     })
     .await
-    .unwrap();
```

Comment:
A `Future` created by `actix_web::web::block` will result in an `Err` only when the inner closure (`run_upload` in this case) panics, which isn't expected, so keep the unwrap here, and handle the result returned by `run_upload`, which is currently ignored.